pontoon/api/util.py

```diff
@@ -0,0 +1,54 @@
+from graphql.language.ast import FragmentSpread
+
+
+def get_fields(info):
```

Comment:
Do you plan to add tests in this PR? I can try to help

Reply:
Yes, I'm working on it!